ggshield/scan/scannable.py

```diff
@@ -107,14 +108,27 @@ def files(self) -> Dict[str, File]:
     def scannable_list(self) -> List[Dict[str, Any]]:
         return [entry.scan_dict for entry in self.files.values()]
 
+    @property
+    def extra_headers(self) -> Dict[str, str]:
+        # get_current_context returns None if outside a click command
+        context = click.get_current_context(silent=True)
+
+        command_path = context.command_path if context is not None else "external"
+        command_name = context.command.name if context is not None else "external"
+
+        return {
+            "GGShield-Version": __version__,
+            "GGShield-Command-Path": command_path,
+            "GGShield-Command-Name": command_name,
```

Comment:
If I understand correctly `command_name` is `command_path[-1]`, if this is correct then maybe we only need `GGShield-Command-Path`?

Reply:
The `command_name` is the name of the function and the `command_path` is the series of commands used in the CLI.   
For example the headers sent by `ggshield scan path setup.py` are:
```json
{
  "GGShield-Version": "1.11.0",
  "GGShield-Command-Path": "ggshield scan path",
  "GGShield-Command-Name": "path-cmd"
}
```

Reply:
OK, but then we can still deduce the command name from the other, and it's more of an implementation detail: we could rename the function and the app would still work the same, as long as the Click decorators are kept unchanged. I think we don't need it.

Reply:
I've removed `GGShield-Command-Name`.

Reply:
Thanks, going to merge it when CI is :green_circle: